<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocalMarkDlg</class>
 <widget class="QDialog" name="LocalMarkDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LocationMark</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLineEdit" name="localNameLedt"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lonLedt"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="localLon">
     <property name="text">
      <string>localLon:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="localName">
     <property name="text">
      <string>localName:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="latLedt"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="localLat">
     <property name="text">
      <string>localLat:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="setApplyBtn">
     <property name="text">
      <string>set</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
